pom.xml

```diff
@@ -390,7 +390,12 @@ under the License.
               <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-compress</artifactId>
-                <version>1.25.0</version>
+                <version>1.26.0</version>
+              </dependency>
+              <dependency>
```

Comment:
If we do this add a comment that this is only needed for commons-compress optional reasons and we should take it out once they fix that.